README.md

```diff
@@ -60,6 +60,57 @@ Please follow the given steps for listing out your company's job opportunities.
   ]
 }
 ```
+## List your profile for Meet The Talent
+In a time of economic uncertainty, with widespread news of layoffs and a tightening job market, our 'Meet the Talent' feature is designed to create connections within the community. Here, we spotlight talented professionals actively seeking opportunities, giving companies access to a diverse pool of potential hires. This initiative is our way of 'connecting the dots' between companies on the lookout for skilled talent and candidates eager to make their mark.
+
+We welcome candidates from the community to share your profile.
+
+Disclaimer: "Meet the Talent" is a community-driven platform connecting job seekers with potential employers. We do not verify the information provided; candidates and companies are responsible for their own due diligence and any agreements made.
+
+**How you can submit your profile for Meet The Talent**
+Please follow the given steps for listing out your profile.
+
+1. Fork the repository
+2. Create a JSON object in the format given below
+3. Add your Job openings JSON object [here](/public/data/meet-the-talent-data.json)
+3. Update the `id` (i.e. last element id + 1)
```

Comment:
Update the readme as we have updated to index rather id

Reply:
Covered in [4f903e4](https://github.com/GDGAhmedabad/devfestahm/pull/359/commits/4f903e4d0b6ea5a43f00e2093a0ee971260c9391) commit